importer.py

```diff
@@ -41,6 +40,7 @@ def __init__(self, db_config_class, collection_name):
         self.image_client = ImageClient(config=db_config_class)
         self.attachment_utils = AttachmentUtils(self.specify_db_connection)
```

Comment:
Importer defines unique MD5  temp jpg path on each init to avoid different jobs from clashing on same tree.